Makefile

```diff
@@ -14,6 +14,22 @@ test: build-all-scenarios
 		--ignore=./smarts/env/tests/test_learning.py \
 		-k 'not test_long_determinism'
 
+.PHONY: sanity-test
+sanity-test: #build-all-scenarios
+	./tests/test_setup.py
+	PYTHONHASHSEED=42 pytest -v \
+		--doctest-modules \
```

Comment:
Added `--junitxml="sanity_test_result.xml"` and mentioned it in the readme. 

Since we only have 8 tests currently it might be fine in my opinion to run all tests and user can show us all the failing tests when they ask for help. Otherwise we might be going back and forth after solving each failing test.